Off-Top-App/off-top-flutter
# Settings page

OT-300: 
Test:
1. get flutter running
2. navigate to the settings page after logging in
3. test the profile and the settings tabs 
4.  make sure they do not overlap when clicked
